erts/emulator/Makefile.in

```diff
@@ -371,6 +371,14 @@ endif
 EPCRE_LIB = $(ERL_TOP)/erts/emulator/pcre/obj/$(TARGET)/$(TYPE)/$(LIB_PREFIX)epcre$(LIB_SUFFIX)
 DEPLIBS += $(EPCRE_LIB)
 
+DEPLIBS += $(RYU_LIBRARY)
+ifeq ($(TARGET),win32)
+LIBS += -L$(RYU_OBJDIR) -lryu
+else
+# Build on darwin fails if -lryu is used
```

Comment:
I think this branch execute for all targets that are not win32 so the comment about darwin is strange. Perhaps it is only Windows that need -lryu? What is the reason for that in that case?

Reply:
I think i simply copied and tweaked the line above for zlib. I am not sure i understand it either. Happy to simplify

Reply:
I think you can remove this comment as I think it causes more confusion than clarity. It seems like it is only Windows that needs special treatment and the other platforms can be handled in the same way.